lib/utils/findFontFamily.mjs

```diff
@@ -27,6 +28,12 @@ function joinValueNodes(firstNode, secondNode, charactersBetween) {
 	return firstNode;
 }
 
+/**
+ * @param {Node} valueNode
+ * @returns {valueNode is postcssValueParser.DivNode}
+ */
+const isCommaDiv = (valueNode) => valueNode.type === 'div' && valueNode.value === ',';
```

Comment:
note

We have 3 rules that have the `foo.type === 'div' && foo.value === ','` condition.
no-duplicate-at-import-rules, declaration-block-no-redundant-longhand-properties and color-function-notation

Reply:
Can you open a refactor issue for this, please.